docs/hub/pro.md

```diff
@@ -0,0 +1,12 @@
+# The HF PRO subscription 🔥
+
+The PRO subscription unlocks essential features for serious users, including:
+
+- Higher [storage capacity](./storage-limits) for public and private repositories
```

Comment:
```suggestion
- More [storage capacity](./storage-limits) for public and private repositories
```
To avoid repeating Higher across both the points